components/music/lists/OffsetTrackList.tsx

```diff
@@ -30,15 +30,15 @@ const OffsetTrackList = ({
         {tracks.map((track, index) => (
           <tr className="hover" key={track.id}>
             {showNumber && <td>{index + 1}</td>}
-            <td>
+            <td className="truncate max-w-[400px]">
               <a
                 className="font-bold text-primary"
                 href={`/app/track/${track.id}`}
               >
                 {track.name}
               </a>
             </td>
-            <td>
+            <td className="truncate max-w-[400px]">
```

Comment:
rather than arbitrarily truncating (which won't work for smaller screen sizes), set a max-w on the div wrappign the entire container.